.github/workflows/go.yml

```diff
@@ -0,0 +1,60 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build + Test Stable
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: auto
+    steps:
+
+    - name: Build
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.x'
```

Comment:
I think this needs to be `[1.17.x]`, otherwise it could run _any_ Go `1.x` version, not the latest.

Reply:
Maybe possible to test with the 3 latest versions even? 🙂

Reply:
Why? They are EOL.